customext/cflush.cc

```diff
@@ -10,17 +10,17 @@ struct : public arg_t {
 
 static reg_t custom_cflush(processor_t* p, insn_t insn, reg_t pc)
 {
+  require_accelerator;
   require_privilege(PRV_M);
-
-  return pc + 4; \
+  return pc + 4;
 }
 
 class cflush_t : public extension_t
 {
  public:
   const char* name() { return "cflush"; }
 
-  cflush_t() {}
+  cflush_t() { always_dirty = true; }
```

Comment:
This should be passed in as a constructor param, instead of allowing the parent class to set `always_dirty` incorrectly and then changing it here. Same for class `dummy_rocc_t`.